projects/packages/my-jetpack/_inc/components/connection-status-card/index.tsx

```diff
@@ -142,15 +142,23 @@ const getUserConnectionLineData: getUserConnectionLineDataType = ( {
 		};
 	}
 
+	const userConnectionText = userConnectionData.currentUser?.wpcomUser?.display_name
+		? sprintf(
+				/* translators: first placeholder is user name, second is either the (Owner) string or an empty string */
+				__( 'Connected as %1$s%2$s.', 'jetpack-my-jetpack' ),
+				userConnectionData.currentUser?.wpcomUser?.display_name,
+				userConnectionData.currentUser?.isMaster ? __( ' (Owner)', 'jetpack-my-jetpack' ) : ''
+		  )
+		: sprintf(
+				/* translators: first placeholder is either the (Owner) string or an empty string */
+				__( 'User connected%1$s.', 'jetpack-my-jetpack' ),
+				userConnectionData.currentUser?.isMaster ? __( ' (Owner)', 'jetpack-my-jetpack' ) : ''
```

Comment:
I know this is how the string was until now, but I think it would be nice to have a single string instead of a single ` (Owner)` that is very hard to translate on its own. Do you think you could change that logic so the strings include ` (Owner)` when possible?